WordPress/Classes/ViewRelated/Reader/ReaderSearchViewController.swift

```diff
@@ -20,6 +20,25 @@ import Gridicons
             case .sites: return NSLocalizedString("Sites", comment: "Title of a Reader tab showing Sites matching a user's search query")
             }
         }
+
+        var trackingValue: String {
+            switch self {
+            case .posts: return "posts"
+            case .sites: return "sites"
+            }
+        }
+    }
+
+    private enum SearchSource {
+        case userInput
+        case searchHistory
+
+        var value: String {
+            switch self {
+            case .userInput: return "user_input"
+            case .searchHistory: return "search_history"
+            }
+        }
     }
```

Comment:
Could this be simplified a little to:

```
private enum SearchSource: String {
    case userInput = "user_input"
    case searchHistory = "search_history"
}
```

and then use `rawValue` where we were using `value`?

Reply:
That's a way better idea than what I had. Updated the code: 035d8dc2415357528a7aa89a222a0b575e5f7c77